doc/configtables/sector.csv

```diff
@@ -63,6 +63,7 @@ tes_tau,,,The time constant used to calculate the decay of thermal energy in the
 -- decentral,days,float,The time constant in decentralized thermal energy storage (TES)
 -- central,days,float,The time constant in centralized thermal energy storage (TES)
 boilers,--,"{true, false}",Add option for transforming electricity into heat using resistive heater

```

Comment:
maybe we can also update this documentation bit in the same go, because this is not electricity -> heat, but gas -> heat, no?
```suggestion
boilers,--,"{true, false}",Add option for transforming gas into heat using gas boilers

```